packages/api/src/functions/graphql.ts

```diff
@@ -14,7 +14,8 @@ import {
 
 export type GetCurrentUser = (
   decoded: AuthContextPayload[0],
-  raw: AuthContextPayload[1]
+  raw: AuthContextPayload[1],
+  options?: AuthContextPayload[2]
```

Comment:
I was looking at the auth code just now and see that in the `decoders` we call this `req`:

```js
interface Req {
  event: APIGatewayProxyEvent
  context: GlobalContext & LambdaContext
}
```

see: https://github.com/redwoodjs/redwood/blob/main/packages/api/src/auth/decoders/index.ts#L14

So, perhaps a better name for this than `options` is `req`? As in `req` and `res` for request and response which is how the handlers [typically call that](https://docs.aws.amazon.com/lambda/latest/dg/nodejs-handler.html).

Reply:
Love it. Updated in all the spots.